openshift-client-api/src/main/java/io/fabric8/openshift/client/dsl/OpenShiftMonitoringAPIGroupDSL.java

```diff
@@ -32,8 +32,8 @@
 import io.fabric8.openshift.api.model.monitoring.v1.ServiceMonitorList;
 import io.fabric8.openshift.api.model.monitoring.v1.ThanosRuler;
 import io.fabric8.openshift.api.model.monitoring.v1.ThanosRulerList;
-import io.fabric8.openshift.api.model.monitoring.v1alpha1.AlertmanagerConfig;
-import io.fabric8.openshift.api.model.monitoring.v1alpha1.AlertmanagerConfigList;
```

Comment:
Shall we add an entry for deletion of `v1alpha1` group in https://github.com/fabric8io/kubernetes-client/blob/main/doc/MIGRATION-v7.md#model-changes ?